api/v1/invitations/PKPInvitationController.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+/**
+ * @file api/v1/invitations/PKPInvitationController.php
+ *
+ * Copyright (c) 2023 Simon Fraser University
+ * Copyright (c) 2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPInvitationController
+ *
+ * @ingroup api_v1_invitations
+ *
+ * @brief Controller class to handle API requests for invitations operations.
+ *
+ */
+
+namespace PKP\API\v1\invitations;
+
+use APP\facades\Repo;
+use Exception;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use PKP\core\PKPBaseController;
+use PKP\core\PKPRequest;
+use PKP\invitation\invitations\contacts\IApiHandleable;
+use PKP\invitation\invitations\PKPCreateInvitationController;
+use PKP\invitation\invitations\PKPInvitation;
+use PKP\invitation\invitations\PKPReceiveInvitationController;
+use PKP\invitation\models\Invitation;
+use PKP\security\Role;
+
+class PKPInvitationController extends PKPBaseController
+{
+    public const PARAM_TYPE = 'type';
```

Comment:
I kept those three 

```
public const PARAM_TYPE = 'type';
public const PARAM_ID = 'invitationId';
public const PARAM_KEY = 'key';
```

and refactored the exception messages so that they don't use the 'Type', 'Id' etc keywords.